src/libappimage/libappimage.cpp

```diff
@@ -28,27 +28,25 @@
 #endif
 
 using namespace appimage::core;
+using namespace appimage::utils;
 
 namespace bf = boost::filesystem;
 
-appimage::utils::Logger libappimageLogger("libappimage", std::clog);
-
 extern "C" {
 
 
 /* Check if a file is an AppImage. Returns the image type if it is, or -1 if it isn't */
 int appimage_get_type(const char* path, bool verbose) {
     typedef std::underlying_type<AppImageFormat>::type utype;
-
     try {
         AppImage appImage(path);
         return static_cast<utype>(appImage.getFormat());
     } catch (const std::runtime_error& err) {
         if (verbose)
-            libappimageLogger.error() << " at " << __FUNCTION__ << " : " << err.what() << std::endl;
+            Logger::error(std::string(__FUNCTION__) + " : " + err.what());
     } catch (...) {
         if (verbose)
-            libappimageLogger.error() << " at " << __FUNCTION__ << " : that's all we know." << std::endl;
+            Logger::error(std::string(__FUNCTION__) + ": that's all we know.");
```

Comment:
"that's all we know" is a pretty bad wording. As said above, there should be a `Logger::exception` method.

Reply:
There is not  specific exception type here therefore we cannot create a nice log message.

Reply:
"unexpected error" is better than "that's all we know", the latter is not quite professional.